cmd/webhook.go

```diff
@@ -177,4 +212,11 @@ func init() {
 	webhookCmd.Flags().StringVar(&loglevel, "loglevel", "info", "loglevel to use, one of: debug, info, warn, error, dpanic, panic, fatal")
 	webhookCmd.Flags().DurationVar(&certCheckInterval, "check-interval", 5*time.Minute, "certificate check interval")
 	webhookCmd.Flags().DurationVar(&certLookaheadInterval, "lookahead-interval", crds.LookaheadInterval, "certificate check interval")
+	// https://go.dev/blog/tls-cipher-suites explains the ciphers selection process
+	webhookCmd.Flags().StringVar(&tlsCiphers, "tls-ciphers", "", "comma separated list of tls ciphers allowed."+
```

Comment:
Oh my 🤦 my bad. I ended up returning an empty list if the string is empty. This will let the TLS options select a sane and safe default.

Let me know if you prefer your suggestion and want to set the defaults on our side!

Reply:
that works for me, but i believe we must return `nil` if we want to fall back to the safe defaults.